### DISCUSSION TITLE
How to show pdf in vscode in browser

### DISCUSSION THREAD

**Discussion by hello-zhanghao** [gitpod-io/openvscode-server#226] [Q&A]
In the gitpod I can use latex-workshop and show pdf in the vscode in browser, but when I build the openvscode server in my docker containers, I will sess the error 127.0.0.1 refused connect.
![image](https://user-images.githubusercontent.com/38358269/141645294-bf8e66a9-f9ed-4e6d-88e1-2f24d8d0cc79.png)
 The reason is that the latex-workshop used local port forwarding, and it requests local port 127.0.0.1 but  the vscode is run in remote and can not connected by 127.0.0.1. So how does gitpod solve the problem.

**Comment by akosyakov** [comment]
It seems like this extension is not updated to run in remote context. Please reach out to the extension author.
